README.md

```diff
@@ -32,7 +32,10 @@ This template is the result of learnings from many previous projects and should
 
 ### Adjust the template to your needs
 
-- Use this repo [as a template](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) and replace all occurrences of "Greeter" in the relevant CMakeLists.txt with the name of your project
+- Use this repo [as a template](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template).
+- Replace all occurrences of "Greeter" in the relevant CMakeLists.txt with the name of your project
+  - Note that capitalization matters here: "Greeter" means the name of the project, while "greeter" is used in file names!
```

Comment:
As we're talking about code changes I think code markdown may be more appropriate.

```suggestion
  - Capitalization matters here: `Greeter` means the name of the project, while `greeter` is used in file names.
```